### DISCUSSION TITLE
Solidity

### DISCUSSION THREAD

**Discussion by akraskovSMDB** [neuropoly/axondeepseg#634] [Help needed]
Hi,

do you have any recommendations on how to use solidity metric for quality control of the segmentation. Is there any recommended threshold value for solidity metric above which axon and its myelin can be considered "well" segmented?

Thank you

**Comment by hermancollin** [comment]
I think it really depends on your data and on the quality of your segmentations, but I'm not sure solidity alone can be used to perform quality control. I am not entirely familiar with the way the convex hull is computed for the axons, but If you have some elements with slight concave features in your data, they could be very precisely segmented but the solidity will not be close to 1. Conversely, you could have a bad segmentation that happens to be perfectly convex, so its solidity value will be 1 but it will not be any good.

If you happen to see a clear pattern in your data where well segmented axons have higher solidity values that bad ones, it might work. Also, I guess you could potentially identify outliers if you see that solidity is ridiculously low (e.g. I doubt a solidity value below 0.4 could ever come out of a good axonal segmentation, but I could be wrong and again it's data dependent)
